javascript/javascript3/week2/currencyCalculator.js

```diff
@@ -0,0 +1,64 @@
+const currencies = [];
+const ratesStorage = {};
+
+const fromSelect = document.getElementById("from-currency");
+const toSelect = document.getElementById("to-currency");
+
+const inputTag = document.getElementById("from-amount");
+const toConverted = document.getElementById("to-amount");
+
+async function fetchCurrencyRates(base = "EUR") {
+    try {
+        const res = await fetch(`https://open.er-api.com/v6/latest/${base}`);
+        const data = await res.json();
+
+        currencies.push(...Object.keys(data.rates));
```

Comment:
Should currencies be a global variable? Can't you just have this function return them?